ethereum/contracts/BeefyLightClient.sol

```diff
@@ -395,12 +408,11 @@ contract BeefyLightClient {
             "Error: Block wait period not over"
         );
 
-        uint256[] memory randomBitfield =
-            Bitfield.randomNBitsWithPriorCheck(
-                getSeed(data),
-                data.validatorClaimsBitfield,
-                requiredNumOfSignatures
-            );
+        uint256[] memory randomBitfield = Bitfield.randomNBitsWithPriorCheck(
```

Comment:
Lets rather agree on & perform formatting changes in a separate PR.

Reply:
this file has been changed all over anyway, so it's not feasible nor important to worry about whitespace mismatch for it i think.